google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineSupportProvider.java

```diff
@@ -300,6 +302,10 @@ public void addSupport(@NotNull Module module,
       AppEngineSupportProvider.this
           .addSupport(module, rootModel, myFrameworkSupportModel,
               PersistenceApiComboboxUtil.getSelectedApi(myPersistenceApiComboBox));
+
+      UsageTrackerProvider.getInstance()
+          .trackEvent(GctTracking.APP_ENGINE_SUPPORT_ADDED)
```

Comment:
It _should_ track imports if you import an ae standard project and then add framework support by clicking on the popup. 

But regardless, for new project's we should probably be explicit about tracking this action rather than tracking it's side effect.